src/data/watchLaterSlice.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from "@reduxjs/toolkit"
+
+const watchLaterSlice = createSlice({
+    name: 'watch-later',
+    initialState: {
+        watchLaterMovies: []
+    },
+    reducers: {
+        addToWatchLater: (state, action) => {
+            state.watchLaterMovies = [action.payload, ...state.watchLaterMovies]
+        },
+        removeFromWatchLater: (state, action) => {
+            const indexOfId = state.watchLaterMovies.findIndex(key => key.id === action.payload.id)
+            state.watchLaterMovies.splice(indexOfId, 1)
+        },
+        remveAllWatchLater: (state) => {
```

Comment:
There is a typo in the action name "remveAllWatchLater". It should be corrected to "removeAllWatchLater".